core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1800,6 +1800,15 @@ private[spark] object Utils extends Logging {
     }
   }
 
+  lazy val isInInterpreter: Boolean = {
```

Comment:
That's more of an architectural question. Currently this method is used only in ClosureCleaner, correct.